api/cpp/include/slint_models.h

```diff
@@ -25,9 +25,9 @@ struct ModelChangeListener
 using ModelPeer = std::weak_ptr<ModelChangeListener>;
 
 template<typename M>
-auto access_array_index(const std::shared_ptr<M> &model, size_t index)
+auto access_array_index(const std::shared_ptr<M> &model, float index)
```

Comment:
In Rust you're first converting from float to isize explicitly, and then to usize. My feeling is that we should do the same for C++, for clarity of the interface: So take an `ssize_t` here, do the negativity check once, and then continue with `size_t`, but let the *caller* worry about the conversion from float to `ssize_t`.